diff --git a/node_modules/remotion/dist/get-environment.js b/node_modules/remotion/dist/get-environment.js
index 7a07b7a..63f9feb 100644
--- a/node_modules/remotion/dist/get-environment.js
+++ b/node_modules/remotion/dist/get-environment.js
@@ -3,7 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.getRemotionEnvironment = void 0;
 const getRemotionEnvironment = () => {
     if (process.env.NODE_ENV === 'production') {
-        if (typeof window !== 'undefined' && window.remotion_isPlayer) {
+        if (global.remotion_isPlayer || (typeof window !== 'undefined' && window.remotion_isPlayer)) {
             return 'player-production';
         }
         return 'rendering';
@@ -14,7 +14,7 @@ const getRemotionEnvironment = () => {
     if (process.env.NODE_ENV === 'test') {
         return 'rendering';
     }
-    if (typeof window !== 'undefined' && window.remotion_isPlayer) {
+    if (global.remotion_isPlayer || (typeof window !== 'undefined' && window.remotion_isPlayer)) {
         return 'player-development';
     }
     return 'preview';
